.github/workflows/yamllint.yml

```diff
@@ -0,0 +1,16 @@
+---
+## Source: https://github.com/marketplace/actions/yaml-lint
+
+name: Yaml Lint
+on: [push, pull_request]  # yamllint disable-line rule:truthy
```

Comment:
To avoid this error Github Actions support an alternative syntax:

```
"on":
  pull_request:
  push:
```

Reply:
I've already tested that without `"`: https://github.com/aalaesar/install_nextcloud/pull/128/commits/475c40323607e2a008fb5274efed4f4257a2d3f5 and there was still a problem with `truthy` error.

This comment based switch was suggested on official repository of yammlint, there is whole thread about it: https://github.com/adrienverge/yamllint/issues/430 
So actually both solutions are workarounds and we just have to wait for this issue being fixed.

I'd leave it as it is right now, because it's not so obvious that adding `"` would fix/broke whole pipeline, but with this comment it's visible at the first glance :)

Reply:
Ah.. double quotes are needed, I am sure of that. The problem is with on.